src/init.cpp

```diff
@@ -391,12 +391,22 @@ void SetupServerArgs()
 
     // Hidden Options
     std::vector<std::string> hidden_args = {
-        "-dbcrashratio", "-forcecompactdb",
-        "-interrupt-block=<hash|height>", "-stop-block=<hash|height>",
-        "-mocknet", "-mocknet-blocktime=<secs>", "-mocknet-key=<pubkey>",
+        "-dbcrashratio",
+        "-forcecompactdb",
+        "-leveldbchecksum=<auto|true|false>",
+        "-interrupt-block=<hash|height>",
+        "-stop-block=<hash|height>",
+        "-mocknet",
+        "-mocknet-blocktime=<secs>",
+        "-mocknet-key=<pubkey>",
         "-checkpoints-file",
         // GUI args. These will be overwritten by SetupUIArgs for the GUI
```

Comment:
All the options below this line can be removed as they applied to the removed Qt GUI.